core/object/script_language.cpp

```diff
@@ -459,7 +461,6 @@ void ScriptServer::remove_global_class_by_path(const String &p_path) {
 bool ScriptServer::is_global_class(const StringName &p_class) {
 	return global_classes.has(p_class);
 }
-
```

Comment:
Please restore, there should be an empty line between methods

Reply:
Thanks for catching that. removed the line while moving a function around. Fixed.